docs/keywords.json

```diff
@@ -1,6 +1,7 @@
 {
   "2d-explore":{
     "alias":[
+      "1.5.0",
```

Comment:
What's the thinking behind adding the version as an alias?

Reply:
We can't search the most recent icons by release number. This helps us know what was added in the current release. We could do something where we added a "new icons" tag to each new release, but this way we don't have to delete those tags in the future.